### PR TITLE
IOD-120 Create Git ACtion for TEST deploy

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

IOD-120

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by SDToews — COMMENTED** [review]
it looks like you've added a test action script and renamed the dev action script to test as well?

File: .github/workflows/dev-app-build-deploy.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-name: DEV Build and Deploy SLAM API
-
+name: TEST Deploy of SLAM Frontend Application Image
```
Comment (akroon3r) [inline .github/workflows/dev-app-build-deploy.yaml:1]:
```suggestion
name: DEV Deploy of SLAM Frontend Application Image 
```

Comment (akroon3r) [inline .github/workflows/dev-app-build-deploy.yaml:1]:
```suggestion
name: DEV Deploy of SLAM Frontend Application Image 
```

**Comment by akroon3r** [comment]
@SDToews whoopsie, fixed that.

**Review by SDToews — APPROVED** [review]

**Pull request merged**